modules/custom-status/custom-status.php

```diff
@@ -41,116 +37,47 @@ class Custom_Status extends Module {
 	const METADATA_REQ_USER_IDS_KEY = 'required_user_ids';
 	const METADATA_REQ_USERS_KEY = 'required_users';
 
-	/**
-	 * Register the module with VIP Workflow but don't do anything else
-	 */
-	public function __construct() {
-
-		$this->module_url = $this->get_module_url( __FILE__ );
-		// Register the module with VIP Workflow
-		$args         = [
-			'module_url'        => $this->module_url,
-			'slug'              => 'custom-status',
-			'configure_page_cb' => 'print_configure_view',
-		];
-		$this->module = VIP_Workflow::instance()->register_module( 'custom_status', $args );
-	}
+	const PUBLISHED_STATUSES = array(
```

Comment:
Minor: Looks like `array()`s have almost entirely been removed from code, regexing `\Warray\(` shows 16 results left. Would you mind removing these as well?

Reply:
Additionally, it looks like `PUBLISHED_STATUSES` is only used in one place in `core-hacks.php` so we might as well move to that file as well.

Reply:
Done all this as well